api/app/Providers/EventServiceProvider.php

```diff
@@ -14,8 +14,8 @@ class EventServiceProvider extends ServiceProvider
      * @var array<class-string, array<int, class-string>>
      */
     protected $listen = [
-        ApplicationSubmitted::class => [
```

Comment:
Don't we need to keep the ApplicationSubmitted event?

Reply:
It looks as though it actually wasn't in use. I'm not actually sure what it was removed though :thinking:

Reply:
Oh, weird! I was a little confused as well. I guess we are not using event listeners to create snapshots anymore? I guess that sort of makes sense since we can just store it in the mutation itself since it can only happen from that one spot :woman_shrugging: